core/domain/feedback_services.py

```diff
@@ -178,6 +183,19 @@ def create_message(
     new_status = thread.status
     thread.put()
 
+    # We do a put on the suggestion linked to the thread, if any so that the
```

Comment:
Perhaps - `We do a put on the suggestion linked (if it exists) to the thread so that the....`

Reply:
Done